app/components/Content/Content.js

```diff
@@ -27,24 +35,48 @@ type Props = {
  * </Content>
  * ```
  */
-function Content({ banner, children, className }: Props) {
-  return (
-    <div>
-      {banner && (
-        <div className={cx(styles.cover, className)}>
-          <Image src={banner} />
+
+class Content extends React.Component<Props, State> {
+  state = {
```

Comment:
A bit unsure if I want to change the `<Content>` component  like this or just change `<ArticleDetail>` instead. This works for now, but I think it's a bit risky to change the `<Content>` component

Reply:
Hmm. Imo its ok, because we can then reuse it in events.

Adding state to the Content component is another case tho.

Reply:
True, but I think I will need state if I want the component to expand if clicked 🤔